backend/app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Http\Controllers;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Hash;
+use App\Models\User;
+use Illuminate\Http\Request;
+
+class UserController extends Controller
+{
+    public function index() {
+        return User::all();
+    }
+    public function store(Request $request) {
```

Comment:
```suggestion
    }
    
    public function store(Request $request) {
```
Add new line after a function